yogo/README.md

```diff
@@ -20,24 +20,3 @@
 | 🎼&#160;Sheetlet&#160;🎼 |   Limited   | Sheetlet displays the music represented as a sheetmusic.                                                     |
 | 🎸&#160;Synthlet&#160;🎸 | **WIP** | Synthlet provides a simple synthesizer to playback the music. It has a dedicated local transport on its own. |
 | 🎹&#160;Rolelet&#160;🎹  | COMING SOON | Rolelet is a DawletScore-compliant piano roll. It accepts an incoming midi to perform a realtime recording.  |
-
-## TODOs
```

Comment:
TODOs will be managed in the [project board](https://github.com/orgs/dawlet-team/projects/1)